src/Ocelot/Configuration/MetadataOptions.cs

```diff
@@ -0,0 +1,45 @@
+﻿using Ocelot.Configuration.File;
+using System.Globalization;
+
+namespace Ocelot.Configuration;
+
+public class MetadataOptions
```

Comment:
It appears to be an exact duplicate of the `FileMetadataOptions` class. What is the purpose of having this separate class?